index.js

```diff
@@ -123,17 +124,18 @@ var compile = function(schema, cache, root, reporter, opts) {
 
     var indent = 0
     var error = function(msg) {
-      if (reporter === false) {
-        validate('errors++')
-        return
+      validate('errors++')
+      if (reporter === true) {
+        validate('if (validate.errors === null) validate.errors = []')
+        if (verbose) {
+          validate('validate.errors.push({field:"%s",message:"%s",value:%s})', formatName(name), msg, name)
```

Comment:
Why is it that `field` is not allways the full path to the additional prop? (Showing the prop that is additional)

Our developers are really confused about this, even though we are using verbose mode.They thought they had an additinal property called `data`.

The `value` is not really a value, more a field name right?